### PR TITLE
LUN-227: Fix cut off header text

### PR DESCRIPTION
**Pull request by f1sh1918**
Fixed header text cut off.

**Test:**
Gastronomie, Lebensmittel -> Bäcker

Tested on android & ios (native and emulator)

### CODE REVIEW

File: src/navigation/Navigator.tsx [hunk 1]
```diff
@@ -28,26 +28,32 @@ export const styles = StyleSheet.create({
...
-    zIndex: 100
-  },
-  headerRight: {
-    paddingRight: 15
+  headerLeftContainer: {
+    flexGrow: 3
   }
 })
 
+export const NavigationTitle = styled.Text`
```
Comment (f1sh1918) [inline src/navigation/Navigator.tsx:40]: Should we put styles in separate file?
I think in this case it make sense to reuse it f.e. in ExerciseHeader

Comment (klinzo) [reply src/navigation/Navigator.tsx:40]: I think it makes sense if it is actually used in multiple places. Then just create a `NavigationTitle` component file

File: src/navigation/Navigator.tsx [hunk 2]
```diff
@@ -140,7 +146,7 @@ const Navigator = (): JSX.Element => {
           options={{
             headerLeft: () => null,
             headerTitle: ' ',
```
Comment (klinzo) [inline src/navigation/Navigator.tsx:148]:
```suggestion
            headerTitle: '',
```

**Review by klinzo — APPROVED** [review]
Tested on iOS simulator

File: src/navigation/Navigator.tsx (hunk 1 above)
Comment (f1sh1918) [reply src/navigation/Navigator.tsx:40]: hm the problem is that this is not a real component but a style wrapper for a component that i want to share.
i thought about sth like this f.e. having a folder like "test" but "styles" and create Navigator inside with its styles
https://blog.cloudboost.io/separate-your-code-with-styled-components-ec4fd1ee3ef8

Comment (klinzo) [reply src/navigation/Navigator.tsx:40]: Its usable as a component though (and technically is a react component), isn't it? So no problem creating a separate file for it :)

**Review by SteffiStuffel — APPROVED** [review]
Looks good. Tested on ios emulator and android emulator.

**Pull request merged**